kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1172,6 +1172,13 @@ object KyuubiConf {
       .checkValues(OperationLanguages.values.map(_.toString))
       .createWithDefault(OperationLanguages.SQL.toString)
 
+  val SESSION_CONF_ADVISOR: OptionalConfigEntry[String] =
+    buildConf("session.conf.advisor")
+      .doc("Choose a custom config advisor for session, the default implementation return empty.")
```

Comment:
let's detail the doc of the config, e.g. what's happening while using this plugin, not just duplicates the config name with some words reordered, besides `the default implementation return empty` is redundant too as we will have a `<td>` for default value.

Reply:
Additionally, we need a separate doc for the plugin. there is nowhere to find the relation between SessionConfAdvisor and this config